opentech/static_src/src/sass/apply/components/_button.scss

```diff
@@ -221,4 +206,53 @@
             fill: $color--white;
         }
     }
+
+    &--action {
+        display: flex;
+        font-weight: $weight--normal;
+        color: $color--default;
+        transition: none;
+
+        @include media-query(tablet-landscape) {
+            background: none;
+            padding: 0;
+            border: 0;
+            align-items: center;
+            justify-content: flex-start;
+            max-width: initial;
```

Comment:
Not sure why this is necessary as max-width not previously set.

Reply:
ah, was refactoring and must have left it in from the filters button. thanks